check.py

```diff
@@ -698,6 +697,32 @@ def execute():
               shutil.copyfile(recreated, 'a.wasm.wast')
               execute()
 
+
+# Run all the tests
+run_help_tests()
+run_wasm_opt_tests()
+s2wasm.test_s2wasm()
```

Comment:
This looks like it changes the order of what gets ran when. I'm not inherently opposed to that, but I'm not sure if the existing order had some broader reason behind it.

Reply:
Oops. Fixed.